k8s/app/deployment-prod.yml

```diff
@@ -34,5 +34,5 @@ spec:
             initialDelaySeconds: 3
             periodSeconds: 5
           env:
-            - name: API_URL
+            - name: REACT_APP_API_URL
```

Comment:
As explained above, using the `REACT_APP_` prefix means it can be used both by:
- node: when this var is present, proxy all the calls to the api to this url
- create-react-app (the frontend): when this var is present, fetch `/api` which is proxied to the docker container (on the node side). If not present, we're in local dev, so fetch `http://localhost:4000/api` which is the local API node server